Changes

```diff
@@ -64,6 +64,16 @@ Working version
   (Enguerrand Decorne, report by Jon Ludlam,
   review by Tom Kelly, KC Sivaramakrishnan and Gabriel Scherer)
 
+- #11082 The runtime now reserves only minor_heap_wsz words of
+  memory per domain for the minor heap area (down from 2GB per domain).
```

Comment:
What is `minor_heap_wsz` ?  Is the reader of the Changes file supposed to know?

Also, `down from 2GB per domain` is inexact since commit aff58350d.